radarpipeline/io/reader.py

```diff
@@ -255,8 +255,12 @@ def _get_field(self, data_type: Union[str, Dict]) -> Any:
         Any
             A Spark data type
         """
-
-        if type(data_type) is str and data_type in self.data_type_mapping:
+        if type(data_type) is list:
+            if len(data_type) == 1:
+                return self.data_type_mapping[data_type[0]]
+            else:
+                return StringType()
```

Comment:
I think we should move this else condition out and use it at line 268 where currently we are raising error through `raise ValueError(f"Unknown data type: {data_type}")`. It would serve as a fallback for all types that the schema reader does not understand.